docs/charts/match_weights_chart.ipynb

```diff
@@ -168,15 +165,46 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### What the chart shows\n"
+    "!!! tip \"Tip\"\n",
+    "\n",
+    "    Make sure to scroll out when hovering over the chart to see the end of each comparison level bar."
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### How to interpret the chart\n"
+    "### What the chart shows\n",
+    "\n",
+    "The `match_weights_chart` show the results of a trained Splink model. Each comparison within a model is represented in a bar chart, with a bar showing the evidence for two records being a match (i.e. match weight) for each comparison level.\n",
+    "\n",
+    "??? note \"What the chart tooltip shows\"\n",
+    "\n",
+    "        ![](./img/match_weights_chart_tooltip.png)\n",
+    "\n",
+    "        The tooltip shows information based on the comparison level bar that the user is hovering over, including:\n",
+    "\n",
+    "        - The name of the comparison and comaprison level.\n",
+    "        - The comparison level condition as an SQL statement.\n",
+    "        - The m and u proability for the comparison level.\n",
+    "        - The resulting bayes factor and match weight for the comparison level."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to interpret the chart\n",
+    "\n",
+    "Each bar in the `match_weights_chart` shows the evidence of a match provided by each level in a Splink model (i.e. match weight). As such, the match weight chart provides a summary for the entire Splink model, as it shows the match weights for every type of comparison defined within the model.\n",
+    "\n",
+    "Any Splink score generated to compare two records will add up the evidence (i.e. match weights) for each comparison to come up with a final match weight score, which can then be converted into a probability of a match.\n",
+    "\n",
+    "The first bar chart is the Prior Match Weight, which is the . This can be thought of in the same way as the y-intercept of a simple regression model\n",
+    "\n",
+    "This chart is an aggregation of the [`m_u_parameters_chart`](./m_u_parameters_chart.ipynb). The match weight for a comparison level is simply $log_2(\\frac{m}{u})$."
```

Comment:
Link to Robin's article for more info?

Reply:
is there a specific article you are thinking of here?